src/tejashree/Assignment8While.java

```diff
@@ -0,0 +1,63 @@
+package tejashree;
+
+public class Assignment8While {
+
+	void printEven(int start, int end) {
+		int index = start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?